apps/src/gamelab/GameLabSprite.js

```diff
@@ -105,6 +108,58 @@ module.exports.createSprite = function (x, y, width, height) {
     }
   });
 
+  Object.defineProperty(s, 'width', {
+    enumerable: true,
+    get: function () {
+      if (width === undefined) {
+        return 100;
+      } else if (!s.animation) {
+        return unscaledWidth;
+      } else {
+        console.log("old width: " + width);
```

Comment:
Here's another reference to `width`.  I think I figured this out - it's the argument passed to `createSprite` that we were modifying as a local - so we could change it, but any internal changes p5.play made to it would not be picked up.